<launch>
   <include file="$(find rcvm_aqua)/launch/rcvm.launch"/>
	<node pkg="rcvm_aqua" type="indicate_object_right.sh" name="indicate_object"/>
</launch>

